### PR TITLE
fix(checkbox): add scaling to ios alert and checkbox

### PR DESCRIPTION
**Pull request by thetaPC**
Issue number: N/A

---------

## What is the current behavior?

Checkboxes inside alerts were not scaling.

## What is the new behavior?

- Checkboxes inside alerts are scaling.
- Switched from `math.div` to `calc` since there is a warning when using rem: `math.div() will only support number arguments in a future release.`
- Found the correct max size for checkboxes on iOS
- Dug deeper into the size for checkboxes on Android. I couldn't find a native example but did find an example for radio. Radios do not scale at all. I'm assuming that checkboxes follow that same logic. So checkboxes on Android will not scale.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

A max width example can be found on the Health app on iOS.

### CODE REVIEW

**Comment by liamdebeasi** [comment]
Synced the changes in. Re-running screenshots now

File: core/src/components/checkbox/checkbox.ios.vars.scss
Comment (thetaPC) [inline core/src/components/checkbox/checkbox.ios.vars.scss:1]: Reverting to not scale. I dug deeper into the size for checkboxes on Android. I couldn't find a native example but did find an example for radio. Radios do not scale at all. I'm assuming that checkboxes follow that same logic. So checkboxes on Android will not scale.

**Comment by thetaPC** [comment]
@liamdebeasi okay, now it's ready for review. Thank you!

File: core/src/components/checkbox/checkbox.ios.vars.scss
Comment (liamdebeasi) [reply core/src/components/checkbox/checkbox.ios.vars.scss:1]: Thanks for catching this! Can confirm that checkboxes don't scale either: 
image

Mind updating the design doc for this when you get a chance?

File: core/src/components/checkbox/checkbox.ios.vars.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@ $checkbox-ios-background-color-on:        ion-color(primary, base) !default;
 $checkbox-ios-background-color-focused:   ion-color(primary, tint) !default;
 
 /// @prop - Size of the checkbox icon
-$checkbox-ios-icon-size:                  dynamic-font-max(26px, 2.38) !default;
+$checkbox-ios-icon-size:                  dynamic-font-max(26px, 2.535) !default;
```
Comment (liamdebeasi) [inline core/src/components/checkbox/checkbox.ios.vars.scss:18]: Why did this change?

Comment (thetaPC) [reply core/src/components/checkbox/checkbox.ios.vars.scss:18]: I was able to get PixelSnap installed and found that the checkboxes are actually bigger. The max is `66px`.

File: core/src/components/checkbox/checkbox.ios.vars.scss
Comment (thetaPC) [reply core/src/components/checkbox/checkbox.ios.vars.scss:1]: Created a PR.

**Review by amandaejohnston — APPROVED** [review]

File: core/src/components/checkbox/checkbox.ios.vars.scss (hunk 1 above)
Comment (sean-perkins) [reply core/src/components/checkbox/checkbox.ios.vars.scss:18]: If the max is `66px`, shouldn't this be `2.538`?

Comment (thetaPC) [reply core/src/components/checkbox/checkbox.ios.vars.scss]: Done

File: core/src/components/checkbox/checkbox.md.vars.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@ $checkbox-md-background-color-focused:   ion-color(primary, tint) !default;
 $checkbox-md-icon-background-color-on:   ion-color(primary, base) !default;
 
 /// @prop - Size of the checkbox icon
-$checkbox-md-icon-size:                  dynamic-font(18px) !default;
+$checkbox-md-icon-size:                  18px !default;
```
Comment (liamdebeasi) [inline core/src/components/checkbox/checkbox.md.vars.scss:20]: Can we add a code comment that explains why we do not use dynamic-font here?

**Review by liamdebeasi — APPROVED** [review]

File: core/src/components/checkbox/checkbox.md.vars.scss (hunk 2 above)
Comment (thetaPC) [reply core/src/components/checkbox/checkbox.md.vars.scss:22]: done

**Review by liamdebeasi — APPROVED** [review]
Thank you!

**Pull request merged**